addons/project/controllers/portal.py

```diff
@@ -222,6 +226,40 @@ def portal_my_project_subtasks(self, project_id, task_id, page=1, date_begin=Non
             pager_vals['url_args'].update(filterby=filterby)
             pager = portal_pager(**pager_vals)
 
+            values.update({
+                'project': project_sudo,
+                'show_project': True,
+                'task': task_sudo,
+                'grouped_tasks': values['grouped_tasks'](pager['offset']),
+                'pager': pager,
+                'searchbar_filters': OrderedDict(sorted(searchbar_filters.items())),
+                'filterby': filterby,
+            })
+            return request.render("project.portal_my_tasks", values)
+        except (AccessError, MissingError):
+            return request.not_found()
+
+    @http.route('/my/projects/<int:project_id>/task/<int:task_id>/recurrent_tasks', type='http', auth='user', methods=['GET'], website=True)
+    def portal_my_project_recurrent_tasks(self, project_id, task_id, page=1, date_begin=None, date_end=None, sortby=None, filterby=None, search=None, search_in='content', groupby=None, **kw):
+        try:
+            project_sudo = self._document_check_access('project.project', project_id)
+            task_sudo = request.env['project.task'].search([('project_id', '=', project_id), ('id', '=', task_id)], limit=1).sudo()
+            task_domain = [('id', 'in', task_sudo.recurrence_id.task_ids.ids)]
+            searchbar_filters = self._get_my_tasks_searchbar_filters([('id', '=', project_id)], task_domain)
+
+            if not filterby:
+                filterby = 'all'
+            domain = searchbar_filters.get(filterby, searchbar_filters.get('all'))['domain']
+
+            values = self._prepare_tasks_values(
+                page, date_begin, date_end, sortby, search, search_in, groupby,
+                url=f'/my/projects/{project_id}/task/{task_id}/recurrent_tasks',
+                domain=AND([task_domain, domain])
+            )
+            values['page_name'] = 'project_recurrent_tasks'
+            # pager
```

Comment:
Either the line is explicit enough, or the comment is not explicit enough, but in either case I don't think it's useful